# 4 — Three emails

A ranked list is an artefact. **An email is the job.**

Each of these is written to one of the top three accounts and cites a specific fact drawn from the
data about *that company* — not a merge field, and not a fact that would be true of any developer.
The test each has to pass: **could this email have been sent to anyone else on the list?** If yes, it
is a mail merge and it belongs in the bin.

All three follow the same shape, which is the shape of a discovery call compressed into six lines:
observation the recipient can verify, the inference that observation supports, the cost of that
inference being true, and one low-commitment ask.

In [1]:
import pandas as pd

import config

top = pd.read_csv(config.OUTPUT_DIR + "/top_25.csv").head(3)
top[["CompanyName", "AgeYears", "AccountsCategory", "MortgagesOutstanding",
     "GroupCompanies", "Score"]]

,CompanyName,AgeYears,AccountsCategory,MortgagesOutstanding,GroupCompanies,Score
0,SIBNER GROUP LTD,4.3,GROUP,3,2,100.0
1,DSBL HMO PORTFOLIO LIMITED,6.2,GROUP,4,32,100.0
2,KAJIMA STUDENT HOUSING LIMITED,7.4,FULL,5,16,98.4


---

## 1 — Sibner Group Ltd · score 100.0

**What the data says:** incorporated May 2022 (4.3 years), registered SW6, files **GROUP** accounts,
**3 outstanding charges** across a 2-company family.

**Why that is worth an email:** a four-year-old company filing *consolidated* accounts has already
outgrown a single entity, and three live charges on a group that small means the balance sheet is
working hard. That combination — young, structurally complex, capital-committed — is the point at
which land sourcing stops being one person's spreadsheet.

> **Subject: three live facilities, four years in**
>
> Hi [name],
>
> Sibner has been filing group accounts since well before most four-year-old developers bother —
> and Companies House shows three charges still outstanding across the group.
>
> Running that much development finance this early usually means site sourcing is still whoever has
> time that week, rather than a process. The developers we work with hit that wall right about here:
> the deal flow justifies a second acquisitions hire before the reporting does.
>
> Worth fifteen minutes on how [competitor-type] teams your size shortened the time from
> off-market listing to offer?
>
> [signature]

---

## 2 — DSBL HMO Portfolio Limited · score 100.0

**What the data says:** incorporated June 2020 (6.2 years), Berkeley Street W1J, **GROUP** accounts,
**4 outstanding charges**, and **32 companies** in the family.

**Why that is worth an email:** thirty-two entities for an HMO portfolio is not a coincidence, it is
an operating model — one SPV per property or per finance package. The commercial pain is not sourcing,
it is that any question spanning the portfolio has to be reassembled by hand from thirty-two sets of
records.

> **Subject: 32 entities, one portfolio view**
>
> Hi [name],
>
> DSBL is now 32 companies on the register, four with charges outstanding. For an HMO portfolio that
> usually means one SPV per property and one finance package per SPV.
>
> The teams we work with tell us the structure is fine until someone asks a portfolio-level question
> — where the next three acquisitions should go, which postcodes are already saturated — and the
> answer takes a week of spreadsheet archaeology across every entity.
>
> If that sounds familiar, I can show you what the portfolio view looks like in about ten minutes.
>
> [signature]

---

## 3 — Kajima Student Housing Limited · score 98.4

**What the data says:** incorporated April 2019 (7.4 years), St Giles Square WC2H, **FULL** accounts,
**5 outstanding charges**, **16 companies** in the family.

**Why that is worth an email:** purpose-built student accommodation is site selection under unusually
hard constraints — the scheme only works within walking distance of a university with a housing
shortfall. Five live charges across sixteen entities says a pipeline is in flight, and PBSA site
selection is a data problem before it is a construction one.

> **Subject: PBSA site selection, five schemes in**
>
> Hi [name],
>
> Kajima Student Housing has five charges outstanding across sixteen entities — a pipeline in flight
> rather than a single scheme.
>
> PBSA is the hardest site-selection problem in UK development: the catchment either works or it
> doesn't, and by the time a site is openly marketed the maths usually doesn't. Most of the teams we
> work with are trying to get to sites before they are listed, using planning history and ownership
> data rather than agent particulars.
>
> Happy to walk through how that is done for student schemes specifically — fifteen minutes?
>
> [signature]

---

## Why these are not mail merges

Apply the test. Email 1 turns on *group accounts unusually early*. Email 2 turns on *32 entities*.
Email 3 turns on *PBSA catchment economics*. Swap any one onto a different recipient and the opening
line becomes false — which is the whole point.

Every fact used is free, public, and machine-readable at segment scale. Nothing here required a data
provider, a scraper or a subscription; the entire personalisation budget was one monthly CSV.

## What would come next

This is where the pipeline stops and a CRM starts. The honest next steps, in order:

1. **A named contact.** Companies House publishes officers and persons with significant control —
   not in this bulk file, but in a companion download. That is the obvious next enrichment.
2. **Send them, and record what happens.** Reply rate by score decile is the only thing that will show
   whether the weights in notebook 2 are right. Until then they are argued, not evidenced — and
   saying so is more useful than pretending otherwise.
3. **Re-point the config.** The same engine, aimed at construction SIC codes or at 62010, produces a
   different segment for a different employer in about ten minutes. Nothing above is specific to
   property developers except the reasoning, and the reasoning is the part worth rewriting.